ci/playbooks/tasks/set_crc_insecure_registry.yml

```diff
@@ -12,14 +12,15 @@
     }
     }' image.config.openshift.io/cluster
 
-- name: Patch the image.config.openshift.io resource to allow insecure registry
+- name: Patch the image.config.openshift.io resource to allow secure/insecure registry
```

Comment:
This task define only secure registries :)

Reply:
`Patch the image.config.openshift.io resource to allow registries` would be good imho, if we really want to edit that task name :).

Reply:
++

Reply:
Done